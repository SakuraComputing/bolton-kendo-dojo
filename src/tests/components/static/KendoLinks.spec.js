import React from 'react'
import KendoLinks   from '../../../components/static/KendoLinks';
import { shallow } from 'enzyme';


describe('Kendo Links', () => {
    it('should render correctly', () => {
        const wrapper = shallow(<KendoLinks />)
        expect(wrapper).toMatchSnapshot();
    })
})