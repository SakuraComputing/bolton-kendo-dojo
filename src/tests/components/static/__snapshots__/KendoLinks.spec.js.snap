// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Kendo Links should render correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <KendoLinks />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          Kendo Links
        </h1>,
        <h2>
          Kendo Federations
        </h2>,
        <p />,
        <a
          href="http://www.britishkendoassociation.com"
          rel="noopener noreferrer"
          target="_blank"
        >
          British Kendo Association
        </a>,
        <p />,
        <a
          href="http://www.ekf-eu.com/"
          rel="noopener noreferrer"
          target="_blank"
        >
          European Kendo Federation
        </a>,
        <p />,
        <a
          href="https://www.kendo-fik.org/english-page/english-top-page.html"
          rel="noopener noreferrer"
          target="_blank"
        >
          All Japan Kendo Federation
        </a>,
        <h2>
          Books
        </h2>,
        <a
          href="https://www.amazon.co.uk/Kendo-Definitive-Guide-Hiroshi-Ozawa/dp/4770021194"
          rel="noopener noreferrer"
          target="_blank"
        >
          Kendo: The Defintive Guide, Hiroshi Ozawa
        </a>,
        <p />,
        <a
          href="https://www.amazon.co.uk/Kendo-Comprehensive-Guide-Japanese-Swordsmanship/dp/4805312319/ref=pd_lpo_sbs_14_t_0?_encoding=UTF8&psc=1&refRID=B1WFW3JCYGHM5XXMPFB0"
          rel="noopener noreferrer"
          target="_blank"
        >
          Kendo: A Comprehensive Guide to Japanese Swordsmanship, Geoff Salmon
        </a>,
        <p />,
        <a
          href="https://www.amazon.co.uk/Oshu-Kendo-Renmei-European-1885-1974/dp/4907009232/ref=sr_1_3?s=books&ie=UTF8&qid=1520892379&sr=1-3&keywords=Paul+Budden"
          rel="noopener noreferrer"
          target="_blank"
        >
          The Oshu Kendo Renmei: A History of British and European Kendo (1885-1974), Paul Budden
        </a>,
        <h2>
          Kendo Equipment
        </h2>,
        <p />,
        <p>
          Disclaimer: These are suggested sites to purchase kendo equipment from, we are totally impartial to these sites it's merely here to give our members a chance to purchase what they need to carry out the martial arts all these sites have delivered what they offered that's why we know they are all reliable. They are placed in alphabetical order
        </p>,
        <p />,
        <a
          href="http://www.alljapanbudogu.com/"
          rel="noopener noreferrer"
          target="_blank"
        >
          All Japan Budogu
        </a>,
        <p />,
        <a
          href="http://kendostar.co.uk/"
          rel="noopener noreferrer"
          target="_blank"
        >
          KendoStar
        </a>,
        <p />,
        <a
          href="https://www.ninecircles.co.uk/"
          rel="noopener noreferrer"
          target="_blank"
        >
          Nine Circles
        </a>,
      ],
      "className": "kendo-links content-container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Kendo Links",
        },
        "ref": null,
        "rendered": "Kendo Links",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Kendo Federations",
        },
        "ref": null,
        "rendered": "Kendo Federations",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "British Kendo Association",
          "href": "http://www.britishkendoassociation.com",
          "rel": "noopener noreferrer",
          "target": "_blank",
        },
        "ref": null,
        "rendered": "British Kendo Association",
        "type": "a",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "European Kendo Federation",
          "href": "http://www.ekf-eu.com/",
          "rel": "noopener noreferrer",
          "target": "_blank",
        },
        "ref": null,
        "rendered": "European Kendo Federation",
        "type": "a",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "All Japan Kendo Federation",
          "href": "https://www.kendo-fik.org/english-page/english-top-page.html",
          "rel": "noopener noreferrer",
          "target": "_blank",
        },
        "ref": null,
        "rendered": "All Japan Kendo Federation",
        "type": "a",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Books",
        },
        "ref": null,
        "rendered": "Books",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Kendo: The Defintive Guide, Hiroshi Ozawa",
          "href": "https://www.amazon.co.uk/Kendo-Definitive-Guide-Hiroshi-Ozawa/dp/4770021194",
          "rel": "noopener noreferrer",
          "target": "_blank",
        },
        "ref": null,
        "rendered": "Kendo: The Defintive Guide, Hiroshi Ozawa",
        "type": "a",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Kendo: A Comprehensive Guide to Japanese Swordsmanship, Geoff Salmon",
          "href": "https://www.amazon.co.uk/Kendo-Comprehensive-Guide-Japanese-Swordsmanship/dp/4805312319/ref=pd_lpo_sbs_14_t_0?_encoding=UTF8&psc=1&refRID=B1WFW3JCYGHM5XXMPFB0",
          "rel": "noopener noreferrer",
          "target": "_blank",
        },
        "ref": null,
        "rendered": "Kendo: A Comprehensive Guide to Japanese Swordsmanship, Geoff Salmon",
        "type": "a",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "The Oshu Kendo Renmei: A History of British and European Kendo (1885-1974), Paul Budden",
          "href": "https://www.amazon.co.uk/Oshu-Kendo-Renmei-European-1885-1974/dp/4907009232/ref=sr_1_3?s=books&ie=UTF8&qid=1520892379&sr=1-3&keywords=Paul+Budden",
          "rel": "noopener noreferrer",
          "target": "_blank",
        },
        "ref": null,
        "rendered": "The Oshu Kendo Renmei: A History of British and European Kendo (1885-1974), Paul Budden",
        "type": "a",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Kendo Equipment",
        },
        "ref": null,
        "rendered": "Kendo Equipment",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Disclaimer: These are suggested sites to purchase kendo equipment from, we are totally impartial to these sites it's merely here to give our members a chance to purchase what they need to carry out the martial arts all these sites have delivered what they offered that's why we know they are all reliable. They are placed in alphabetical order",
        },
        "ref": null,
        "rendered": "Disclaimer: These are suggested sites to purchase kendo equipment from, we are totally impartial to these sites it's merely here to give our members a chance to purchase what they need to carry out the martial arts all these sites have delivered what they offered that's why we know they are all reliable. They are placed in alphabetical order",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "All Japan Budogu",
          "href": "http://www.alljapanbudogu.com/",
          "rel": "noopener noreferrer",
          "target": "_blank",
        },
        "ref": null,
        "rendered": "All Japan Budogu",
        "type": "a",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "KendoStar",
          "href": "http://kendostar.co.uk/",
          "rel": "noopener noreferrer",
          "target": "_blank",
        },
        "ref": null,
        "rendered": "KendoStar",
        "type": "a",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Nine Circles",
          "href": "https://www.ninecircles.co.uk/",
          "rel": "noopener noreferrer",
          "target": "_blank",
        },
        "ref": null,
        "rendered": "Nine Circles",
        "type": "a",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            Kendo Links
          </h1>,
          <h2>
            Kendo Federations
          </h2>,
          <p />,
          <a
            href="http://www.britishkendoassociation.com"
            rel="noopener noreferrer"
            target="_blank"
          >
            British Kendo Association
          </a>,
          <p />,
          <a
            href="http://www.ekf-eu.com/"
            rel="noopener noreferrer"
            target="_blank"
          >
            European Kendo Federation
          </a>,
          <p />,
          <a
            href="https://www.kendo-fik.org/english-page/english-top-page.html"
            rel="noopener noreferrer"
            target="_blank"
          >
            All Japan Kendo Federation
          </a>,
          <h2>
            Books
          </h2>,
          <a
            href="https://www.amazon.co.uk/Kendo-Definitive-Guide-Hiroshi-Ozawa/dp/4770021194"
            rel="noopener noreferrer"
            target="_blank"
          >
            Kendo: The Defintive Guide, Hiroshi Ozawa
          </a>,
          <p />,
          <a
            href="https://www.amazon.co.uk/Kendo-Comprehensive-Guide-Japanese-Swordsmanship/dp/4805312319/ref=pd_lpo_sbs_14_t_0?_encoding=UTF8&psc=1&refRID=B1WFW3JCYGHM5XXMPFB0"
            rel="noopener noreferrer"
            target="_blank"
          >
            Kendo: A Comprehensive Guide to Japanese Swordsmanship, Geoff Salmon
          </a>,
          <p />,
          <a
            href="https://www.amazon.co.uk/Oshu-Kendo-Renmei-European-1885-1974/dp/4907009232/ref=sr_1_3?s=books&ie=UTF8&qid=1520892379&sr=1-3&keywords=Paul+Budden"
            rel="noopener noreferrer"
            target="_blank"
          >
            The Oshu Kendo Renmei: A History of British and European Kendo (1885-1974), Paul Budden
          </a>,
          <h2>
            Kendo Equipment
          </h2>,
          <p />,
          <p>
            Disclaimer: These are suggested sites to purchase kendo equipment from, we are totally impartial to these sites it's merely here to give our members a chance to purchase what they need to carry out the martial arts all these sites have delivered what they offered that's why we know they are all reliable. They are placed in alphabetical order
          </p>,
          <p />,
          <a
            href="http://www.alljapanbudogu.com/"
            rel="noopener noreferrer"
            target="_blank"
          >
            All Japan Budogu
          </a>,
          <p />,
          <a
            href="http://kendostar.co.uk/"
            rel="noopener noreferrer"
            target="_blank"
          >
            KendoStar
          </a>,
          <p />,
          <a
            href="https://www.ninecircles.co.uk/"
            rel="noopener noreferrer"
            target="_blank"
          >
            Nine Circles
          </a>,
        ],
        "className": "kendo-links content-container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Kendo Links",
          },
          "ref": null,
          "rendered": "Kendo Links",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Kendo Federations",
          },
          "ref": null,
          "rendered": "Kendo Federations",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "British Kendo Association",
            "href": "http://www.britishkendoassociation.com",
            "rel": "noopener noreferrer",
            "target": "_blank",
          },
          "ref": null,
          "rendered": "British Kendo Association",
          "type": "a",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "European Kendo Federation",
            "href": "http://www.ekf-eu.com/",
            "rel": "noopener noreferrer",
            "target": "_blank",
          },
          "ref": null,
          "rendered": "European Kendo Federation",
          "type": "a",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "All Japan Kendo Federation",
            "href": "https://www.kendo-fik.org/english-page/english-top-page.html",
            "rel": "noopener noreferrer",
            "target": "_blank",
          },
          "ref": null,
          "rendered": "All Japan Kendo Federation",
          "type": "a",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Books",
          },
          "ref": null,
          "rendered": "Books",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Kendo: The Defintive Guide, Hiroshi Ozawa",
            "href": "https://www.amazon.co.uk/Kendo-Definitive-Guide-Hiroshi-Ozawa/dp/4770021194",
            "rel": "noopener noreferrer",
            "target": "_blank",
          },
          "ref": null,
          "rendered": "Kendo: The Defintive Guide, Hiroshi Ozawa",
          "type": "a",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Kendo: A Comprehensive Guide to Japanese Swordsmanship, Geoff Salmon",
            "href": "https://www.amazon.co.uk/Kendo-Comprehensive-Guide-Japanese-Swordsmanship/dp/4805312319/ref=pd_lpo_sbs_14_t_0?_encoding=UTF8&psc=1&refRID=B1WFW3JCYGHM5XXMPFB0",
            "rel": "noopener noreferrer",
            "target": "_blank",
          },
          "ref": null,
          "rendered": "Kendo: A Comprehensive Guide to Japanese Swordsmanship, Geoff Salmon",
          "type": "a",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "The Oshu Kendo Renmei: A History of British and European Kendo (1885-1974), Paul Budden",
            "href": "https://www.amazon.co.uk/Oshu-Kendo-Renmei-European-1885-1974/dp/4907009232/ref=sr_1_3?s=books&ie=UTF8&qid=1520892379&sr=1-3&keywords=Paul+Budden",
            "rel": "noopener noreferrer",
            "target": "_blank",
          },
          "ref": null,
          "rendered": "The Oshu Kendo Renmei: A History of British and European Kendo (1885-1974), Paul Budden",
          "type": "a",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Kendo Equipment",
          },
          "ref": null,
          "rendered": "Kendo Equipment",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Disclaimer: These are suggested sites to purchase kendo equipment from, we are totally impartial to these sites it's merely here to give our members a chance to purchase what they need to carry out the martial arts all these sites have delivered what they offered that's why we know they are all reliable. They are placed in alphabetical order",
          },
          "ref": null,
          "rendered": "Disclaimer: These are suggested sites to purchase kendo equipment from, we are totally impartial to these sites it's merely here to give our members a chance to purchase what they need to carry out the martial arts all these sites have delivered what they offered that's why we know they are all reliable. They are placed in alphabetical order",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "All Japan Budogu",
            "href": "http://www.alljapanbudogu.com/",
            "rel": "noopener noreferrer",
            "target": "_blank",
          },
          "ref": null,
          "rendered": "All Japan Budogu",
          "type": "a",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "KendoStar",
            "href": "http://kendostar.co.uk/",
            "rel": "noopener noreferrer",
            "target": "_blank",
          },
          "ref": null,
          "rendered": "KendoStar",
          "type": "a",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Nine Circles",
            "href": "https://www.ninecircles.co.uk/",
            "rel": "noopener noreferrer",
            "target": "_blank",
          },
          "ref": null,
          "rendered": "Nine Circles",
          "type": "a",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
